app/templates/gentelella/admin/menu.html

```diff
@@ -4,9 +4,25 @@
             <nav>
                 <ul class="nav navbar-nav">
                     <li style="margin-left: 10px;">
-                        <a href="/" style="font-size: 26px; font-weight: 300;">
-                            {{ app_name }}
-                        </a>
+
+                            {% set vars = {'foo': False,img: 'dummy'} %}
+                                {% for custom_image in custom_placeholder %}
+                                    {% if custom_image.name == "Logo" %}
```

Comment:
It would be awesome if you can figure something out ... We could use the same for all other placeholder images too. I feel the current implementation is very hacky.